source/releasenotes/2025-01-15-drupal-cms-upstream.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Drupal CMS upstream now available
+published_date: "2025-01-15"
+categories: [drupal, new-feature]
+---
+
+[Drupal CMS](https://new.drupal.org/drupal-cms) is available to install on Pantheon. Just launched today by the Drupal Association, Drupal CMS is a Drupal 11 variant that combines powerful, user-friendly content creation and management tools with robust flexibility and scalability for developers and end-users alike.
+
+For installation details, check out the new [Drupal CMS docs](/supported-drupal#drupal-cms-on-pantheon). Please join us in the [Pantheon Community Slack](https://docs.pantheon.io/pantheon-community) to celebrate the release!
+
+Our support team is ready and available if you have any questions or issues with installation. 
```

Comment:
I think a note at the end that says something about installing via the dashboard is coming would be good here

Reply:
Added a note